build.sbt

```diff
@@ -199,5 +198,6 @@ lazy val `mauth-jvm-clients` = (project in file("."))
     publishSettings,
     crossScalaVersions := Nil,
     publish / skip := false,
-    mimaPreviousArtifacts := Set.empty
+    mimaPreviousArtifacts := Set.empty,
```

Comment:
I think this one if from the mima plugin that was removed

Reply:
removed, now it's handled by `smartReleaseAggregateProjectSettings`